pages/common/git-range-diff.md

```diff
@@ -0,0 +1,16 @@
+# git range-diff
+
+> Compare two commit ranges (e.g. two versions of a branch).
+> More information: <https://git-scm.com/docs/git-range-diff>.
+
+- Diff the changes of two individual commits:
+
+`git range-diff {{commit_1}}^! {{commit_2}}^!`
+
+- Diff the changes of ours and theirs from their common ancestor, e.g. after an interactive rebase:
+
+`git range-diff {{theirs}}...{{ours}}`
+
+- Diff the changes of two commit ranges, e.g. to check whether conflicts have been resolved appropriately when rebasing commits from `base1` to `base2`:
+
+`git range-diff {{base1}}..{{rev1}} {{base2}}..{{rev2}}`
```

Comment:
```suggestion
- Diff the changes of two commit ranges, e.g. to check whether conflicts have been resolved appropriately when rebasing commits from `base_1` to `base_2`:

`git range-diff {{base_1}}..{{rev_1}} {{base_2}}..{{rev_2}}`
```

Reply:
I'm not sure that this is necessary — the extra underscores render it a little less readble IMO.

Reply:
I also think we should leave it. Since neither `e`, nor `v` is full-height, the number following it is quite readable.

Reply:
However to keep standard, the underscores were used earlier

Reply:
I guess we should revert that, since git itself also uses no underscores, see for example:

    find /usr/share/man -name "*git*" | xargs gunzip -c | grep commit1

produces some results while the same with underscores doesn't

Reply:
Sometimes underscores (aka snake_case) make it more readable, and it other situations they don't.

In this case, given the number of items here I'm not convinced that this improves it.

Reply:
feel free to ignore this then, just keeping standard since snake was used earlier

Reply:
https://github.com/tldr-pages/tldr/pull/5701/files#diff-83dbb0d1c631c1f5de07d28506db0756b728f046436d4ae0524d7f05d46c2364R8

For reference to what my previous message included.